ret-core/src/main/kotlin/io/rabobank/ret/IntelliSearch.kt

```diff
@@ -23,6 +23,13 @@ data class FilterRound(val filter: String, val startIndex: Int, val endIndex: In
     fun hasFailed() = endIndex <= 0
 }
 
+/**
+ * Match results like IntelliJ (to be injected)
```

Comment:
What does "to be injected" mean (in this context)?

Reply:
"to be injected" in the sense that plugin creators need to inject this class if they want to use it (not create one themselves). But probably the "@ApplicationScoped" annotation should suffice for that

Reply:
Yeah, would say so. It doesn't add much to the docs to mention "to be injected".